usaspending_api/awards/management/commands/helpers.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models import Agency, Location, RefCountryCode
+
+
+def up2colon(input_string):
+    'Takes the part of a string before `:`, if any.'
+
+    return input_string.split(':')[0].strip()
+
+
+def get_agency(self, agency_string):
+    agency_code = up2colon(agency_string)
+    agency = Agency.objects.filter(
+        subtier_agency__subtier_code=agency_code).first()
+    if not agency:
+        self.logger.error("Missing agency: " + agency_string)
```

Comment:
self won't work here, also logger is not defined. Might want to pass in the logger or grab it when the function is called